dot/rpc/subscription/listeners.go

```diff
@@ -280,9 +287,31 @@ func (l *RuntimeVersionListener) Listen() {
 	ver.TransactionVersion = rtVersion.TransactionVersion()
 	ver.Apis = modules.ConvertAPIs(rtVersion.APIItems())
 
-	l.wsconn.safeSend(newSubscriptionResponse("state_runtimeVersion", l.subID, ver))
+	go l.wsconn.safeSend(newSubscriptionResponse("state_runtimeVersion", l.subID, ver))
+
+	// listen for runtime updates
+	go func() {
+		for info := range l.runtimeUpdate {
```

Comment:
```suggestion
		for info, ok := range l.runtimeUpdate {
		    if !ok {
		        return
		    }
		    
		    // rest of code
```

add closed channel check

Reply:
When I try to add ok assignment as above, I get compile error `too many variables in range`, is there another way to check this?

Reply:
oh nvm I missed that this is a range, you'd have to change it to read from the channel in a for loop
```
for {
    info, ok := <-l.runtimeUpdate {
       if !ok {
           return
       }
    //rest of code
}
```

Reply:
just pinging on this

Reply:
updated.